src/Helper/Scripts.php

```diff
@@ -116,36 +111,39 @@ public function addCondInternal($cond, $script, $pos = 100)
     /**
      * Adds internal script
      *
-     * @param int $pos The script position in the stack.
+     * @param int   $pos  The script position in the stack.
+     * @param array $attr The additional attributes
      *
-     * @return Scripts
+     *
+     * @return void
      *
      * @access public
      */
-    public function beginInternal($pos = 100)
+    public function beginInternal($pos = 100, array $attr = array())
     {
-        $this->capture[] = $pos;
+        $this->capture[] = array($pos, $attr);
          ob_start();
     }
 
     /**
-     * beginInternalCond
+     * Begin Conditional Internal Capture
      *
-     * @param mixed $cond DESCRIPTION
-     * @param int   $pos  DESCRIPTION
+     * @param mixed $cond condition
+     * @param int   $pos  position
+     * @param array $attr The additional attributes
      *
-     * @return mixed
+     * @return void
```

Comment:
I was looking at some of the aura repos. It looks we have return null for Aura.Di , but there are return void for Aura.Html eg : Styles.php class and also Aura.View have one instance for AbstractView.php . These are quick searches via https://github.com/auraphp/Aura.Di/search?utf8=%E2%9C%93&q=%22%40return+null%22&type=Code . But mostly it is return null.